includes/payment-methods/class-wc-stripe-upe-payment-method-klarna.php

```diff
@@ -20,7 +20,6 @@ public function __construct() {
 		$this->is_reusable                  = false;
 		$this->supported_currencies         = [ 'AUD', 'CAD', 'CHF', 'CZK', 'DKK', 'EUR', 'GBP', 'NOK', 'NZD', 'PLN', 'SEK', 'USD' ];
 		$this->supported_countries          = [ 'AU', 'AT', 'BE', 'CA', 'CZ', 'DK', 'FI', 'FR', 'GR', 'DE', 'IE', 'IT', 'NL', 'NZ', 'NO', 'PL', 'PT', 'ES', 'SE', 'CH', 'GB', 'US' ];
-		$this->accept_only_domestic_payment = true;
```

Comment:
Because Klarna has complex rules around currencies and technically allows cross border transactions (like France to Norway), I've removed this flag. Currency and location rules will be enforced via checkout billing country validation on the checkout.

Reply:
Maybe we could keep the flag as `false` and add this information there for the future.

Reply:
Sounds good. `$accept_only_domestic_payment` is technically false by default but being explicit and including a comment to explain the complexity here is worth the redundancy. 

https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3235/commits/3bb8df02cfd3a20c02691ef0d76c9b83f83396ed